tests/units/signertest.cpp

```diff
@@ -0,0 +1,24 @@
+//#
+//# Copyright (C) 2020-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "signertest.h"
```

Comment:
Add the QtDebug module for using the qWarning macro.
```suggestion
#include "signertest.h"
#include <QDebug>
```